change/@ni-nimble-components-18b7e8bd-7ece-49a0-a662-2a8424f6bfdd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Interesting that hovering over the label causes the control to enter the hover state and that clicking it behaves the same as clicking the control. This was slightly unexpected to me (I thought maybe I could select and copy the label text, but that's not possible). However it matches the behavior of labels on text and number field so I think it's the right call.

No action needed, just observing in case someone else sees this and finds it unexpected like I did.

https://github.com/user-attachments/assets/63e44712-1c99-44f1-9d22-c2013139e6b0

Reply:
Looks like in chrome and firefox a labelled select gets focused when the label is clicked (instead of open) but hover state on the label does apply to control and label text is selectable in control: https://jsbin.com/viveyitene/2/edit?html,js,output

Made a separate comment about selection of label, hover seems aligned, don't feel strongly about click opening vs just focusing

Reply:
Documented the current inconsistencies in #2331